gui/src/themes/stylekit.ts

```diff
@@ -173,6 +175,16 @@ export const stylekitModeThemes = {
             },
         },
         components: {
+            // Give popover paper a slighly lighter color to reflect superior elevation
+            MuiPopover: {
+                styleOverrides: {
+                    paper: {
+                        backgroundColor: !!window?.taipyConfig?.stylekit?.colorPaperLight 
```

Comment:
!!window?. => window.

Reply:
@FredLL-Avaiga Woops, indeed, that was overkill. Fixed.